Cargo.toml

```diff
@@ -33,10 +33,10 @@ version = "0.2.0"
 
 [dependencies.tokio-native-tls]
 optional = true
-version = "0.1"
+version = "0.2"
 
 [dev-dependencies]
 futures-channel = "0.3"
-tokio = { version = "0.2", default-features = false, features = ["io-std", "macros", "stream", "time"] }
+tokio = { version = "0.3", default-features = false, features = ["io-std", "macros", "rt-multi-thread", "stream", "time"] }
```

Comment:
Is `rt-multi-thread` really necessary? This feature will bring more transient dependencies to libraries

Reply:
yes, it's required for as `dev-dependency` for the tests.

Reply:
Hahahahahaha :D

These are `dev-dependencies`. Sorry for the fuzz

Reply:
no problem, tokio 0.3 changed a bit in its `macros` & runtime handling
This is the error you get now
```
error: The #[tokio::test] macro requires rt or rt-multi-thread.
 --> tests/handshakes.rs:4:1
  |
4 | #[tokio::test]
  | ^^^^^^^^^^^^^^
```